bftengine/src/bftengine/FullNodeReplica.cpp

```diff
@@ -0,0 +1,295 @@
+// Concord
+//
+// Copyright (c) 2018, 2019 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").  You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#include <optional>
+#include <functional>
+#include <bitset>
+
+#include <bftengine/Replica.hpp>
+#include <messages/StateTransferMsg.hpp>
```

Comment:
why these brackets? This is our code, same for both lines.
`#include "bftengine/Replica.hpp"
#include "messages/StateTransferMsg.hpp"`

In addition, no need to have the line 19 seperator.